docs/release-notes/.FSharp.Compiler.Service/9.0.300.md

```diff
@@ -1,12 +1,9 @@
 ### Fixed
 
-* Fix Realsig+ generates nested closures with incorrect Generic  ([Issue #17797](https://github.com/dotnet/fsharp/issues/17797), [PR #17877](https://github.com/dotnet/fsharp/pull/17877))
-
 ### Added
 
-
 ### Changed
-
+* Update `Obsolete` attribute checking to account for `DiagnosticId` and `UrlFormat` properties. ([PR #18224](https://github.com/dotnet/fsharp/pull/18224))
 
 ### Breaking Changes
```

Comment:
Just a heads up - is this move on purpose?

Reply:
No. It was a bad merge when fixing conflicts. Fixed now